src/browser.js

```diff
@@ -1,21 +1,37 @@
-const {ipcRenderer} = require('electron');
-const User = require('./User');
-
+/**
+ * @file Render process entry point for Maker Toolkit Browser.
+ * Creates/binds browser controls, including the webview that hosts Code.org
+ * curriculum.
+ *
+ * Process: Renderer
```

Comment:
I started commenting in files on whether they are using the Main or Renderer process, or both, as the Electron docs themselves do.  It helps keep track of where everything's taking place since we sort of have four layers:

* Electron Main process (main.js entry point)
* Electron Renderer process (index.html and browser.js entry point)
* Webview preload script (Webview renderer process, runs in context of document in webview but has access to Node and Electron APIs)
* Webview browser context (Stuff Code.org javascript can see)